tests/baselines/reference/contextuallyTypedStringLiteralsInJsxAttributes02.errors.txt

```diff
@@ -104,7 +102,7 @@ tests/cases/conformance/types/contextualTypes/jsxAttributes/file.tsx(36,44): err
     
     export function NoOverload1(linkProps: LinkProps): JSX.Element { return undefined }
     const d1 = <NoOverload1 {...{goTo:"home"}} extra  />;  // goTo has type "home" | "contact"
-                                               ~~~~~
-!!! error TS2322: Type '{ extra: true; goTo: "home"; }' is not assignable to type 'IntrinsicAttributes & LinkProps'.
-!!! error TS2322:   Property 'extra' does not exist on type 'IntrinsicAttributes & LinkProps'.
+                            ~~~~~~~~~~~~~~~~~~
+!!! error TS2322: Type 'string' is not assignable to type '"home" | "contact"'.
+!!! related TS6500 tests/cases/conformance/types/contextualTypes/jsxAttributes/file.tsx:13:5: The expected type comes from property 'goTo' which is declared here on type 'IntrinsicAttributes & LinkProps'

```

Comment:
Yeah, most of this looks pretty good now, it's just like this one and some similar ones that look a bit questionable compared to the old error to me now.